src/telliot_feed_examples/cli.py

```diff
@@ -165,5 +171,55 @@ def report(
         _, _ = asyncio.run(legacy_reporter.report())
 
 
+@cli.command()
+@click.option(
+    "--amount-trb",
+    "-trb",
+    "amount_trb",
+    help="amount to tip in TRB for a query ID",
+    nargs=1,
+    type=float,
+    required=True,
+)
+@click.pass_context
+def tip(
+    ctx: Context,
+    amount_trb: float,
+) -> None:
+    """Tip TRB for a selected query ID"""
+    legacy_id = ctx.obj["LEGACY_ID"]
+
+    click.echo(f"Tipping {round(amount_trb, 2)} TRB for legacy ID {legacy_id}.")
```

Comment:
suggest removing rounding to avoid confusion in logs.  (test shows tip of 0, even when it is not)

Reply:
Good point, done